src/navigation/pathnavigator_lua.inl

```diff
@@ -307,6 +307,44 @@ namespace {
     }
 }
 
+/**
+ * Fade rendering to black, jump to the specified node, and then fade in. This is done by
+ * triggering another script that handles the logic.
+ *
+ * If no fade duration is specified, use the property from Navigation Handler ->
+ * Path Navigator
+ */
+[[codegen::luawrap]] void jumpTo(std::string nodeIdentifier,
+                                 std::optional<double> fadeDuration)
+{
+    using namespace openspace;
+    if (!sceneGraphNode(nodeIdentifier)) {
```

Comment:
```suggestion
    if (SceneGraphNode* n = sceneGraphNode(nodeIdentifier);  !n) {
```

Reply:
Curious: what's the motivation behind this?

Reply:
Trying to limit the amount of "work" that is done inside the if statement.  Moving from 1. calling `sceneGraphNode` and 2. testing the result against `nullptr` by making it two separate steps